data-raw/watershed/regional_approximation/Regional_Approximation.Rmd

```diff
@@ -0,0 +1,168 @@
+---
+title: "Regional Approximation"
+author: "[Sadie Gill](mailto:sgill@flowwest.com), [Mark Tompkins](mailto:mtompkins@flowwest.com), [Erin Cain](mailto:ecain@flowwest.com)"
+date: "Aug 8, 2019"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
+knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
+library(tidyverse)
+library(readxl)
+library(cvpiaHabitat)
+library(purrr)
+library(lubridate)
+```
+
+
+## Upper-mid Sacramento Region 
+
+### Instream Spawning and Rearing Habitat
+
+**Details:**    
+A mean regional weighted usable area (WUA) was developed for watersheds without instream spawning and rearing habitat modeling. 
+
+For unmodeled tributaries of the Upper Sacramento River, the rearing WUA values were estimated using the mean WUA at each flow from Battle Creek, Butte Creek, Clear Creek, and Cow Creek. The spawning WUA values were estimated using the mean WUA at each flow from Battle Creek, Butte Creek, and Clear Creek.
+
+**Spawning Not Modeled:** 
+
+* Antelope Creek
+* Bear Creek
+* Big Chico Creek
+* Cow Creek
+* Deer Creek
+* Elder Creek
+* Mill Creek
+* Paynes Creek
+* Stony Creek
+* Thomes Creek
+
+**Rearing Not Modeled:**    
+
+* Antelope Creek
+* Bear Creek
+* Big Chico Creek
+* Deer Creek
+* Elder Creek
+* Mill Creek
+* Paynes Creek
+* Stony Creek
+* Thomes Creek
+
+### Combine Spawning and Rearing Data 
+The instream spawning and rearing habitat data described above for Fall Run Chinook Salmon (FR) is combined for use in the DSM in the following format. Units are in square feet per 1000 feet. 
+
+```{r, include = FALSE}
+# Find where modeling exists in upper_mid_sac_region(see ~cvpiaHabitat/data-raw/modeling_exists )
+cvpiaHabitat::modeling_exist %>%
```

Comment:
when you do your review of modeling that exists plz update the `cvpiaHabitat::modeling_exist` data set to reflect these changes